src/main/kotlin/org/ionproject/integration/model/internal/timetable/Event.kt

```diff
@@ -0,0 +1,11 @@
+package org.ionproject.integration.model.internal.timetable
+
+data class Event(
+    @Transient var title: String = "",
```

Comment:
Why the `@Transient`?

Reply:
I-On Core mentioned that `title` property is optional given that we aren't able to obtain that data from the ISEL timetable pdf. With `@Transient` the field isn't serialized.
Another option would be to make optional fields nullable and set as null when not avaible. 
Is that preferable to this?